assisted-installer.yaml

```diff
@@ -1,24 +1,24 @@
 openshift/assisted-installer:
   revision: c81363a7fada8d23cfbac86ecc0f97835f1fa043
   images:
-  - assisted-installer-controller
-  - assisted-installer
+  - quay.io/ocpmetal/assisted-installer-controller
+  - quay.io/ocpmetal/assisted-installer
 openshift/assisted-service:
   revision: b9ab5d3c77580ab93c29bb43cb78cc6b02d54e88
   images:
-  - assisted-service
+  - quay.io/edge-infrastructure/assisted-service
 openshift-assisted/assisted-ui:
   revision: d60808caecfdb4f6c480dd8e5b07213ea4f210dd
   images:
-  - ocp-metal-ui
+  - quay.io/ocpmetal/ocp-metal-ui
 openshift/assisted-installer-agent:
   revision: 1b51ecd2786c0f56ceb8f2051fe0af4dae1f7173
   images:
-  - assisted-installer-agent
+  - quay.io/ocpmetal/assisted-installer-agent
```

Comment:
This one is in edge-infrastructure as well.

Reply:
right
fixed!